AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -170,7 +171,13 @@ public class CardComponent: CardPublicKeyConsumer,
         formViewController.title = paymentMethod.name
         return formViewController
     }()
-
+    
+    // MARK: - ClearableComponent
+    
+    /// :nodoc:
+    public func clear() {
```

Comment:
This PR only fixes `CardComponent`, right?

Reply:
Yeah, the card component for now.